### ISSUE TITLE
modifying from classification to regression

### ISSUE THREAD

**Issue by DecentMakeover** [yhenon/pytorch-retinanet#120]
@yhenon I want to add another regression loss for every box.I have modified the losses.py as such 

            targets = torch.ones(erosion.shape) * -1
            targets = targets.cuda()
            targets[torch.lt(IoU_max, 0.4), :] = 0
            positive_indices = torch.ge(IoU_max, 0.5)
            num_positive_anchors = positive_indices.sum()
            assigned_annotations = bbox_annotation[IoU_argmax, :]
            targets[positive_indices, :] = 0
            targets[assigned_annotations[positive_indices, 5].long()] = 1
            criterion = nn.MSELoss()
            loss = torch.sqrt(criterion(targets, erosion))
            erosion_losses.append(loss)

Does this look okay ? i mean i am not getting any errors but not sure if this will work.

i have modified models.py on line 182 as such 

        self.erosionScore = ClassificationModel(256, num_classes=1)

**Comment by DecentMakeover** [comment]
Fixed
